code-challenges.test.js

```diff
@@ -21,9 +21,53 @@ const fibLength1 = 6
+        // an expect method, that is nested in the test block, calling on the inputNum function, followed by the .toEqual() matcher that checks that the output of the function return 
+
+describe("inputNum", () => {
+        it("to take in a number that is greater than 2 and return an array that length containing the number of the Fibonacci sequence", () => {
+         expect(inputNum(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+         expect(inputNum(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+        })
+})
+
+// Red of red-green refactor: 
+        // inputNum › to take in a number that is greater than 2 and return an array that length containing the number of the Fibonacci sequence
+        // ReferenceError: inputNum is not defined
 
 // b) Create the function that makes the test pass.
 
+// PSEUDOCODE: 
+// Create a function named inputNum 
+// create a variable named newArray to be assigned to an array with the values of [0,1]
+// create a for loop to iterate the amount of times as the agrument to find Fibonacci sequence.
+// Use push() to push the numbers into new array 
+// Use slice() to remove 0 from the array 
+
+const inputNum = (number) => {
+        let newArray = [0,1]
+        for(let i=2; i <= number; i++) {
+                newArray.push(newArray[i-2] + newArray[i-1])
+        }
+        return newArray.slice(1)
+}
+
+console.log(inputNum(fibLength1))
+console.log(inputNum(fibLength2))
+
+// Green of red-green refactor: 
+        // PASS  ./code-challenges.test.js
+        // inputNum
+        //   ✓ to take in a number that is greater than 2 and return an array that length containing the number of the Fibonacci sequence (6 ms)
+
+
+// Notes: 
+// I had some trouble with this prompt. It was difficult to pin point what I needed to do to get the Fibonacci sequence. From reading the prompt, I knoew that I would have to iterate on this number to create the array output so I thought a for loop would be best to use for this situation. However, my blocker was how I could set the for loop to a set number of times I'd like it to loop around. From research online, I learned you can set the second part of the for loop (i < ) and put number - which would allow the function to iterate through the amount of times as the number that was going to be passed through it. 
+
+// My next blocker would be how to do the math needed to have the values of my array correct. With help online, I understood that to create an array, I would need to make another variable to set the array to [0,1]. By having this array, I would be able to utilize it to call for it and do addition math to add the value of the first two elements and then after that, add the sum of the previous two elements. Overall, I was not able to get this answer by myself and needed to use resources found online to get to this. However, now I understand this concept more and can see how I can use it in the future. 
```

Comment:
Good work! That said remember just cause we got a result doesn't mean we can't make it better. and that effort will make it personal!